tasks/Module_2/task_1.py

```diff
@@ -0,0 +1,17 @@
+stuff = {'rope': 1, 'torch': 6, 'gold coin': 42, 'dagger': 1, 'arrow': 12}
+def display_inventory(inventory):
+    print("Inventory:")
+    item_total = 0
+    overload_warning =''
+    for item_name, item_count in inventory.items():
+        item_total += item_count
+        print(str(item_count) + ' ' + item_name)
+    print("\nTotal number of items: " + str(item_total)+'\n')
+    if item_total >= 60 and item_total < 70:
```

Comment:
I think it would be good to introduce constants here.

eg.
MAX_INVENTORY_CAP = 80
HEAVY_WEIGHT_THRESHOLD = 70
LIGHT_WEIGHT_THRESHOLD=60

Feel free to use other naming names if u come up with better ones :D
Also, please double-check the numbers I've provided here, that's just a proposition.

And of course, consts should be moved to the top of the file.

I'd also opt for adding an empty line above line 10 to keep the different parts of the function visually separated.

Reply:
This line with constants should look like `if LIGHT_WEIGHT_THRESHOLD <= item_total < HEAVY_WEIGHT_THRESHOLD`

Reply:
Interesting...  implementation of Between...